cells/presets/nixago.nix

```diff
@@ -0,0 +1,61 @@
+{
+  inputs,
+  cell,
+}: let
+  inherit (inputs) nixpkgs;
+  inherit (inputs.cells) std;
+  l = nixpkgs.lib // builtins;
+in {
+  adrgen = std.nixago.adrgen {
+    configData = import ./nixago/adrgen.nix;
+  };
+  editorconfig = std.nixago.editorconfig {
+    configData = import ./nixago/editorconfig.nix;
+    hook.mode = "copy"; # already useful before entering the devshell
+  };
+  conform = std.nixago.conform {
+    configData = import ./nixago/conform.nix;
+    format = "yaml";
+    output = ".conform.yaml";
```

Comment:
Is this copying over on purpose? If so, I'm not sure I understand why here and not with the others. I'd expect _not_ to redefine it here.